### DISCUSSION TITLE
Feature Request: 独立的微信消息单向同步服务

### DISCUSSION THREAD

**Discussion by Steven-AA** [Quorafind/Obsidian-Thino#581] [Polls 投票]
### Feature Requested | 请求的功能

目前的同步微信消息和thino同步使用同一个开关，必须要开启同步才能收到微信消息。
我只需要单向同步微信消息到ob， 不需要也不想把ob的内容同步到云端，因此希望能有单独的微信消息同步服务。

看目前的计划，thino web 服务还会包含完整的独立web版本，估计后面订阅价格不会很便宜。同步微信消息和完整的web服务相比，功能比较独立，对服务器的负载应该也比较低。 单独需要这个功能的人也许会比需要完整的web服务的人更多些。

抛开订阅的价格，部分用户对于上传笔记内容到服务器会比较敏感。

能否以一个较低的订阅价格提供一个单独的微信同步功能，甚至给pro用户提供 限量的 纯文本 微信消息同步功能。

### Checklist | 检查清单

- [X] The feature would be useful to more users than just me. 该功能对于不仅仅是我自己的用户来说是有用的。

[a comment by an automated account is not shown]

**Comment by juestchaos** [comment]
建议有需求和没有需求的用户都讨论下，这个会影响挺多人的习惯

[a comment by an automated account is not shown]
